lib/jirawatch/cli/track.rb

```diff
@@ -12,21 +12,52 @@ class Track < Dry::CLI::Command
       def call(issue_key:, **options)
         @jira_client.Issue.find(issue_key) # Fails if issue doesn't exist
 
-        puts "Logging time for #{issue_key}..."
         started_at = Time.parse(options.fetch(:started_at, Time.now.to_s))
+        # Time when the timer is paused
+        paused_at = 0
+        # Time when the timer is restarted
+        restarted_at = started_at
+        # Represents all the seconds of actual work spent (it does not count the time of the pauses)
+        partial_time_spent = 0
+
+        puts "Logging time for #{issue_key}..."
         puts started_at
+        puts "\n"
+        puts "Press space to pause/restart the timer"
+        puts "Press enter (or CTRL-X) when you have finished to track the time"
 
-        begin
-          loop do
-            sleep 1
+        reader = TTY::Reader.new
+        reader.on(:keyspace) do
+          if paused_at == 0
+            paused_at = Time.now
+            partial_time_spent = (partial_time_spent + (paused_at - restarted_at)).floor
+
+            # Unit of time printed (the default is always plural)
+            time_unit = "minutes"
+            if (partial_time_spent / 60) == 1
+              time_unit = "minute"
+            end
```

Comment:
That's not very much ruby-like 😄 moreover, the condition is wrong, it should equals to 0.  Please change it to this:
```
time_unit = 
   if partial_time_spent/60 == 0
      "minutes"
   else
      "minute"
   end
```

Reply:
I will change it in ruby like but I think that the condition is correct because I should print `minute` only when 1 minute was tracked. In all other cases I should print `minutes` even when `partial_time_spent` is less than 60 seconds

Reply:
Damn you're right! I was wrong here about the condition :smile: